userguide/subscription/includes/catalog-introduction.adoc

```diff
@@ -1,78 +1,76 @@
+Catalog configuration allows the Kill Bill Administrator to define whether all subscriptions attached to a customer should be invoiced at the same time, or if they should be billed separately on their own schedule. There could also be a mix of both where some are grouped together on the same invoice and others are invoiced independently. In the simplest case, where all subscriptions are grouped together and there is a Bill Cycle Day (BCD) defined (`Billing Period` is a multiple of the month), the customer gets one invoice per `Billing Period` on that day. Note that grouping subscriptions together may by default lead to some proration when the subscription is started on a different date than the BCD, as the system needs to realign invoicing on that particular date. This behavior is configured in the catalog through the http://docs.killbill.io/latest/userguide_subscription.html#_billing_alignment_rules[Billing Alignment Rules].
 
 ==== Creating the Catalog
 
-In this section we look at getting the top level catalog created and configured.
+A Kill Bill catalog, as we know, is an XML document. This section describes how to create a catalog configured to your particular needs.
+
 Note that the catalog currently requires **all attribute strings to be unique**, and those strings **cannot have space**.
 
-The XML document starts in a standard way. The main tag is “catalog”. Within the catalog opening tag you can specify the schema location.
-You can use the standard schema location (above) or you can have the code generate the current schema by executing:
-As an example, the killbill-catalog-x.y.z-xsd-tool.jar for the version `0.13.7` can be found http://search.maven.org/remotecontent?filepath=org/kill-bill/billing/killbill-catalog/0.13.7/killbill-catalog-0.13.7-xsd-tool.jar[here].
+The XML document starts in a standard way. The main tag is `<catalog>`. Within the `catalog` tag you should specify the schema location.
+You can use the standard schema location, as shown in the example, or you can have the code generate the current schema by executing the tool killbill-catalog-x.y.z-xsd-tool.jar, where `x.y.z` is replaced by the appropriate version of Kill Bill. The tool for version `0.13.7` can be found http://search.maven.org/remotecontent?filepath=org/kill-bill/billing/killbill-catalog/0.13.7/killbill-catalog-0.13.7-xsd-tool.jar[here]. After downloading this tool, the schema can be generated by the following command:
```

Comment:
Let's update `0.13.7` with `0.22.0`, which is much more recent 😄 

We also host the generated XSD only in our documentation at http://docs.killbill.io/latest/catalog.xsd. We should add the link as it's more convenient that generating it with the `jar` method.

Also I believe the example should be updated as such:

```
<catalog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="https://docs.killbill.io/latest/catalog.xsd">
```

Reply:
These changes have been made